### ISSUE TITLE
Add support for Adobe Lightroom Classic

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#2502]
[odbol](https://twitter.com/odbol) writes: 

> Neat! I'm still reading about CommandPost architecture: how would it interact with Photoshop? A Lua plugin? Or modifying the core app? Or is there a way to connect Lua cpPlugin with Photoshop's scripting engine?
> 
> I'm also interested in Lightroom support

**Comment by Semmu** [comment]
I would be interested in this, do you need any help?

**Comment by latenitefilms** [comment]
Currently, supporting Lightroom Classic is very much down the list of priorities for us, however, if anyone else wants to take on this project, we'd be more than happy to assist.

You could also use [MIDI2LR](https://github.com/rsjaffe/MIDI2LR/) and use MIDI to communicate between CommandPost and MIDI2LR.

**Comment by Semmu** [comment]
Yes, in the meantime I found midi2lr too and it seems to be exactly what I'm looking for! Thanks for the fast reply!

**Comment by YogieAnamCara** [comment]
I'd love to see Lightroom Classic support too.
